ext/scripts/install_scripts/install_via_r_remotes.pl

```diff
@@ -54,13 +54,57 @@ =head1 SYNOPSIS
 }
 
 
-my $combining_template = "library(remotes)\n<<<<0>>>>";
+my $combining_template = '
+library(remotes)
+install_or_fail <- function(package_name, version){
+
+   tryCatch({install_version(package_name, version, repos="https://cloud.r-project.org", Ncpus=4)
+         library(package_name, character.only = TRUE)},
+         error = function(e){
+             print(e)
+             stop(paste("installation failed for:",package_name ))},
+         warning = function(w){
+           catch <-
+             grepl("download of package .* failed", w$message) ||
+             grepl("(dependenc|package).*(is|are) not available", w$message) ||
+             grepl("installation of package.*had non-zero exit status", w$message) ||
+             grepl("installation of one or more packages failed", w$message)
+           if(catch){ print(w$message)
+             stop(paste("installation failed for:",package_name ))}}
+         )
+
+ }
+
+<<<<0>>>>
+';
+
+my $combining_template_validation = '
```

Comment:
Not sure if we need it, because we already do a library call during the install

Reply:
This function checks if the package is in `available.packages()`.
Point 2 in the [acceptance](https://github.com/exasol/script-languages-release/issues/772#issuecomment-1468173297) criteria of the related ticket.